contrib/bintray/src/mill/contrib/bintray/BintrayPublisher.scala

```diff
@@ -78,7 +78,7 @@ class BintrayPublisher(owner: String,
   private def reportPublishResults(publishResults: Seq[(BintrayPackage, Artifact, Seq[requests.Response])]) = {
     val errors =
       for {
-        (pkg, artifact, responses) <- publishResults
+        (_, artifact, responses) <- publishResults
```

Comment:
Although I can understand your motivation to communicate that it is unused, it makes reasoning about the code harder.
The IDE is able to still detect unused vars, but reading the code immediately makes clear, what the unused variable means. It helps also when changing that tuple later.